Casks/l/lycheeslicer.rb

```diff
@@ -10,7 +10,7 @@
 
   livecheck do
     url "https://mango3d.io/downloads/"
-    regex(/href=.*?LycheeSlicer[._-]v?(\d+(?:\.\d+)+)\.dmg/i)
+    regex(/VERSION:\sv?(\d+(?:\.\d+)+)/i)
```

Comment:
```suggestion
    regex(/version:\s*v?(\d+(?:\.\d+)+)/i)
```

* Technically the text is "version" in the HTML source and is styled as uppercase using a `text-transform: uppercase;` CSS rule. It doesn't matter from a functional standpoint since we use case insensitive regexes but I mention it because it can be important to check the HTML source when creating a regex like this.
* When matching whitespace with `\s`, I generally use `\s*` when optional and `\s+` when required instead of only `\s`. This will allow the regex to continue to match if we end up with more than just one whitespace character (which I've seen happen in some sources over time).